xdp-forward/tests/test-xdp-forward.sh

```diff
@@ -1,7 +1,7 @@
 XDP_LOADER=${XDP_LOADER:-./xdp-loader}
 XDP_FORWARD=${XDP_FORWARD:-./xdp-forward}
-ALL_TESTS="test_ping test_load test_fwd_full test_fwd_direct"
-
+ALL_TESTS="test_ping test_load test_fwd_full test_fwd_direct test_flowtable"
+INFILE="$(mktemp)"
```

Comment:
We already have a `$STATEDIR` that is created in `test_runner.sh`. Let's use that instead of having the test call `mktemp` directly. Maybe it's a good idea to have the test_runner script create a scratch space for each test, and automatically clean it up after the test has run?

So something like this, in `run_tests()`:

```sh
SCRATCH_DIR="${STATE_DIR}/${testn}.scratch"
mkdir "$SCRATCH_DIR"
[... after cleanup]
rm -rf "$SCRATCH_DIR"
```

And then the test itself can just write to `$SCRATCH_DIR/infile`

Reply:
@tohojo ack, I fixed it using `$STATEDIR` directory. I guess we can introduce `$SCRATCH_DIR` in a follow-up patch (it seems cleaner to me)